services-geojson/src/commonMain/kotlin/org/maplibre/geojson/model/Polygon.kt

```diff
@@ -0,0 +1,104 @@
+package org.maplibre.geojson.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import org.maplibre.geojson.serializer.PolygonSerializer
+import org.maplibre.geojson.utils.json
+import kotlin.jvm.JvmStatic
+
+/**
+ * Model representing a GeoJSON Polygon. Polygons are build by an outer [LineString] ring and
+ * zero or more inner [LineString] rings.
+ *
+ * Every ring (inner, and outer) are a closed linear line rings. This means, that minimum four points are required,
+ * and the first and last points must be identical.
+ *
+ * See [Polygon specification](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.6) for more details.
+ *
+ * @param outerLineStringRing the outer [LineString] ring of this Polygon
+ * @param holeLineStringRings the inner [LineString] rings of this Polygon. Or empty list if no inner holes exists
+ * @param boundingBox the BoundingBox of this Polygon
+ * @throws IllegalArgumentException if one of the provided [LineString] is not valid linear rings
+ */
+@Serializable(with = PolygonSerializer::class)
+@SerialName("Polygon")
+data class Polygon(
+    /**
+     * The outer [LineString] ring of this Polygon.
+     */
+    val outerLineStringRing: LineString,
+
+    /**
+     * The inner [LineString] rings of this Polygon. Or empty list if no inner holes exists.
+     */
+    val holeLineStringRings: List<LineString> = emptyList(),
```

Comment:
```suggestion
    /**
     * The outer [LineString] ring of this Polygon.
     */
    val outer: LineString,

    /**
     * The inner [LineString] rings of this Polygon. Or empty list if no inner holes exists.
     */
    val inner: List<LineString> = emptyList(),
```

I think this would be more consistent.